playbook/app/pb_kits/playbook/pb_loading_inline/loading_inline.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { render, screen } from '../utilities/test-utils'
+import LoadingInline from './_loading_inline'
+
+const testId = "loadingInline"
+
+test('should render custom class and data', () => {
+    render(
+        <LoadingInline
+            className='custom-class'
+            data={{ testid: testId }}
+        />
+    )
+
+    const kit = screen.getByTestId(testId)
+    expect(kit).toHaveClass('custom-class')
+})
+
+test('should render id', () => {
+    render(
+        <LoadingInline
+            data={{ testid: testId }}
+            id={testId}
+        />
+    )
+
+    const kit = screen.getByTestId(testId)
+    expect(kit).toHaveProperty('id', testId)
+})
+
+test('should render aria-label', () => {
+    render(
+        <LoadingInline
+            aria={{ label: testId }}
+            data={{ testid: testId }}
+        />
+    )
+
+    const kit = screen.getByTestId(testId)
+    expect(kit).toHaveAttribute('aria-label', testId)
+})
```

Comment:
Looks good! You can probably also add a test for the align prop, just so all the props are covered

Reply:
The way that other kits are testing this case is by checking for the class, and in my opinion, this is a bad practice because it is testing the implementation detail. We agreed to bring this discussion to Jasper, maybe he has another way or we can assume that it is better to test this way, even knowing this is not good, instead of not testing.

Thank you @nidaqg for bringing this out.